docs/persistent_compilation_cache.md

```diff
@@ -8,38 +8,66 @@ when running the same or similar tasks repeatedly.
 
 ## Usage
 
-The compilation cache is enabled when the
-[cache-location](https://github.com/google/jax/blob/jax-v0.4.26/jax/_src/config.py#L1206)
-is set. This should be done prior to the first compilation. Set the location as
-follows:
+### Quick start
 
 ```python
 import jax
+import jax.numpy as jnp
+
+jax.config.update("jax_compilation_cache_dir", "/tmp/jax_cache")
+jax.config.update("jax_persistent_cache_min_entry_size_bytes", -1)
+jax.config.update("jax_persistent_cache_min_compile_time_secs", 0)
 
-# Make sure this is called before jax runs any operations!
-jax.config.update("jax_compilation_cache_dir", "cache-location")
+@jax.jit
+def f(x):
+  return x + 1
+
+x = jnp.zeros((2, 2))
+f(x)
 ```
 
-See the sections below for more detail on `cache-location`.
+### Setting cache directory
+
+The compilation cache is enabled when the
+[cache location](https://github.com/google/jax/blob/jax-v0.4.26/jax/_src/config.py#L1206)
+is set. This should be done prior to the first compilation. Set the location as
+follows:
 
-[`set_cache_dir()`](https://github.com/google/jax/blob/jax-v0.4.26/jax/experimental/compilation_cache/compilation_cache.py#L18)
-is an alternate way of setting `cache-location`.
+(1) Using environment variable
 
-### Local filesystem
+In shell, before running the script:
 
-`cache-location` can be a directory on the local filesystem. For example:
+```sh
+export JAX_COMPILATION_CACHE_DIR="/tmp/jax_cache"
+```
+
+Or on the top of the Python script:
 
 ```python
-import jax
+import os
+os.environ["JAX_COMPILATION_CACHE_DIR"] = "/tmp/jax_cache"
+```
 
-jax.config.update("jax_compilation_cache_dir", "/tmp/jax-cache")
+(2) Using `jax.config.update()`
+
+```python
+jax.config.update("jax_compilation_cache_dir", "/tmp/jax_cache")
 ```
 
-Note: the cache does not have an eviction mechanism implemented. If the
-cache-location is a directory in the local filesystem, its size will continue
-to grow unless files are manually deleted.
+(3) Using [`set_cache_dir()`](https://github.com/google/jax/blob/jax-v0.4.26/jax/experimental/compilation_cache/compilation_cache.py#L18)
+
+```python
+from jax.experimental.compilation_cache import compilation_cache as cc
+cc.set_cache_dir("/tmp/jax_cache")
+```
+
+### Caching thresholds
+
+Only executables that are at least `jax_persistent_cache_min_entry_size_bytes`
+large and take `jax_persistent_cache_min_compile_time_secs` long to compile will
+be cached.
 
-### Google Cloud
+## Placing the cache on GCS
```

Comment:
Nit: I would keep the old section title and level (`###`).

Reply:
Done